detox/test/src/Screens/ActionsScreen.js

```diff
@@ -29,7 +29,7 @@ export default class ActionsScreen extends Component {
         <TouchableOpacity onPress={this.onButtonPress.bind(this, 'Tap Working')}
           onLongPress={this.onButtonPress.bind(this, 'Long Press Working')}
           >
-          <Text style={{ color: 'blue', marginBottom: 20, textAlign: 'center' }}>Tap Me</Text>
+          <Text style={{ color: 'blue', marginBottom: 20, textAlign: 'center' }} acccessibilityLabel={'Tap Me'}>Tap Me</Text>
```

Comment:
I don’t like these changes.
Can you please explain why we need to require users to add `accessibilityLabel` on each element?

On iOS, `accessibilityLabel` is used by Voice Over, for example, and is not always linked 1:1 with the content. The developer can certainly change it if they feel they can provide a better label for the content.
How does this work on Android?

Perhaps we should change the tests on iOS to `by.text(‘’)`, but requiring users to add `accessibilityLabel` on each element is not something I can live with.

Also please note you misspelled “accessibilityLabel” here (“ccc”).